### ISSUE TITLE
Unable to install 1.22.2 upgrade

### ISSUE THREAD

**Issue by AndySteel-kcom** [Microsoft/vscode#48420]
[vscode-inno-updater.log](https://github.com/Microsoft/vscode/files/1938773/vscode-inno-updater.log)
Issue Type: <b>Bug</b>

VSCode 1.22.1 had been running for a few hours / days. Was notified of an upgrade so clicked Update Now. It then proceeded to say that it was installing in the background. I then waited and clicked on "Update Now" when it said it had finished.

The window closed and then a dialog appeared saying I needed to re-install Visual Studio code as it couldn't upgrade.

VS Code version: Code 1.22.1 (950b8b0d37a9b7061b6f0d291837ccc4015f5ecd, 2018-04-06T02:26:57.615Z)
OS version: Windows_NT x64 6.1.7601

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) CPU E5-2630 0 @ 2.30GHz (4 x 2294)|
|Memory (System)|8.00GB (4.53GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|20%|

</details><details><summary>Extensions (13)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|0.19.0
project-manager|ale|0.25.0
vscode-eslint|dba|1.4.8
vscode-stylesheet-formatter|dba|0.2.0
xml|Dot|1.9.2
gitlens|eam|8.2.4
EditorConfig|Edi|0.12.1
dotenv|mik|1.0.1
easy-less|mrc|1.4.5
azure-account|ms-|0.4.0
vscode-docker|Pet|0.0.26
plsql-language|xyz|1.2.2
vsc-docker|Zim|0.34.0

</details>

**Comment by joaomoreno** [comment]
Did you keep a screenshot of that dialog? Did it tell you to send us a log file?

**Comment by AndySteel-kcom** [comment]
No I didn't know I needed to screenshot - assumed it would be in the log file. It did ask me to file an issue and gave me the log file name to upload.

I repeated the attempt to upgrade shortly afterwards and it crashed with a runtime exception and it also mentioned that a setup was already running. In the end I had to kill all the processes, including setup processes before trying again. That time it then worked.
